src/modules/Channel/context/ChannelProvider.tsx

```diff
@@ -421,6 +424,15 @@ const ChannelProvider: React.FC<ChannelContextProps> = (props: ChannelContextPro
       messagesDispatcher,
     },
   );
+  const [sendMultipleFilesMessage] = useSendMultipleFilesMessage({
+    currentChannel: currentGroupChannel,
+    // Open interface to <Channel /> & <ChannelProvider >
+    // onBeforeSendMultipleFilesMessage: (params) => params,
```

Comment:
This too, let me make a different PR and update types

Reply:
Also, we have to give prompt to user if files limit / size validation fails.

Reply:
Yes, it will be included in the other PR